usr/share/rear/prep/default/330_include_uefi_tools.sh

```diff
@@ -1,19 +1,10 @@
 #
-# 310_include_uefi_tools.sh
+# 330_include_uefi_tools.sh
```

Comment:
Do we need to name the script in the script? If nobody sees a reason then let's remove this.

Reply:
I don't see a necessity. Inherited from 6e05b6f0e51fdfdf02c67edddb117b41a364c6f4